prebuilt/boot-assets/config.txt

```diff
@@ -46,6 +46,8 @@ kernel=uboot.bin
 
 dtparam=i2c_arm=on
 dtparam=i2c_vc=on
+#maybe run "sudo modprobe spi-bcm2835" to load kernel modules 
```

Comment:
the comment is not needed just tested , its working fine with the dtparam line alone.

Cheers